ui/test/nuxeo-layout.test.js

```diff
@@ -169,5 +180,81 @@ suite('nuxeo-layout', () => {
       expect(nuxeoInput.validate.calledOnce).to.be.true;
       expect(layout.element.validate.calledOnce).to.be.true;
     });
+
+    test('Should invalidate the layout if nuxeo-file is still loading', async () => {
+      const layout = await fixture(html`
+        <nuxeo-layout href="${base}/layouts/document/file/nuxeo-file-widget-create-layout.html"></nuxeo-layout>
+      `);
+      await layoutLoad(layout);
+
+      // mark the nuxeo-file has performing a file upload
+      const fileInput = layout.element.shadowRoot.querySelector('nuxeo-file');
+      fileInput.uploading = true;
+
+      // the nuxeo-file should be invalid because it is still uploading
+      let validity = await layout.validate();
+      expect(validity).to.be.false;
+      expect(fileInput.invalid).to.be.true;
+
+      // when the file upload finishes, the nuxeo-file should be valid
+      fileInput.uploading = false;
+      validity = await layout.validate();
+      expect(fileInput.invalid).to.be.false;
```

Comment:
should we also include this line as juxtaposition with the above?
```suggestion
      expect(validity).to.be.false;
      expect(fileInput.invalid).to.be.false;
```

Reply:
Sure, it makes sense 👍 Thanks